cdk/src/open-data-platform/data-plane/data-import/data-import-stack.ts

```diff
@@ -40,32 +41,39 @@ export class DataImportStack extends Construct {
       },
```

Comment:
Could you change the timeout back to 5? I'm realizing the timeouts I saw were inside a library in the handler not on the handler itself - thanks!

Reply:
Yup, done.